scarb/src/bin/scarb/args.rs

```diff
@@ -156,7 +157,7 @@ pub enum Command {
     Remove(RemoveArgs),
     /// Compile current project.
     Build(BuildArgs),
-    /// Expand macros.
+    /// Expand proc macros.
```

Comment:
```suggestion
    /// Expand macros.
```
This will actually expand all macros, both proc-* and built-in.